ci-operator/config/redhat-developer/helm/redhat-developer-helm-release-3.11__helm-ocp4.14-lp-interop.yaml

```
@@ -0,0 +1,44 @@
+base_images:
+  base:
+    name: "4.14"
+    namespace: ocp
+    tag: base
+binary_build_commands: make -f openshift-ci/Makefile build
+build_root:
+  project_image:
+    dockerfile_path: openshift-ci/Dockerfile.tests
+images:
+- dockerfile_path: openshift-ci/Dockerfile.tests
+  from: base
+  to: helm
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 200m
+      memory: 200Mi
+tests:
+- as: helm-interop-aws
+  cron: 0 1 * * 1
```

```suggestion
  cron: 0 6 * * 1
```
To keep it consistent with all other interop scenarios, this is a UTC time, which will reflect a 1am run time of EST timezone.

---

sure